Dfe.Data.SearchPrototype/Web/Tests/PageObjectModel/Setup/DocumentObjectModelExtractor.cs

```diff
@@ -2,7 +2,7 @@
 using AngleSharp.Dom;
 using Microsoft.AspNetCore.Mvc.Testing;
 
-namespace DfE.Data.SearchPrototype.Web.Tests.Integration.PageObjectModel.Setup;
+namespace Dfe.Data.SearchPrototype.Web.Tests.PageObjectModel.Setup;
```

Comment:
what's the namespace change for?

Reply:
I've moved the PageObjectModel directory up from the `Integration` dir to the root of the project.